aspnetcore/signalr/dotnet-client.md

```diff
@@ -0,0 +1,73 @@
+---
+title: ASP.NET Core SignalR .NET Client
+author: rachelappel
+description: Information about the ASP.NET Core SignalR .NET Client 
+manager: wpickett
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 05/04/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/dotnet-client
+---
+
+# ASP.NET Core SignalR .NET Client
+
+By [Rachel Appel](http://twitter.com/rachelappel)
+
+The ASP.NET Core SignalR .NET client can be used by Xamarin, WPF, Windows Forms, Console, and .NET Core apps. Like the [JavaScript client](xref:signalr/javascript-client), the .NET client enables you to receive and send messages to a hub in real time.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/live/aspnetcore/signalr/dotnet-client/sample) ([how to download](xref:tutorials/index#how-to-download-a-sample))
+
+The code sample in this article is a WPF client.
+
+## Setup client
+
+The `Microsoft.AspNetCore.SignalR.Client` package is needed for .NET clients to connect to SignalR hubs. To install the client library, run `Install-Package` in the **Package Console Manager** window.
+
+```cmd
+Install-Package Microsoft.AspNetCore.SignalR.Client
+```
+
+## Connect to a hub
+
+To establish a connection, create a `HubConnectionBuilder` and call `Build`. The hub URL, protocol, transport type, log level, headers, and other options can be configured while building a connection. Configure the options by inserting any of the `HubConnectionBuilder` methods into `Build`, such as `WithConsoleLogger`. Start the connection with `StartAsync`.
```

Comment:
What version of SignalR are we targeting because in the RC/RTM bits it's now `ConfigureLogging`